cmake/modules/AddSwift.cmake

```diff
@@ -466,6 +466,11 @@ function(_add_swift_runtime_link_flags target relpath_to_lib_dir bootstrapping)
   if(SWIFT_HOST_VARIANT_SDK IN_LIST SWIFT_DARWIN_PLATFORMS)
 
     set(sdk_dir "${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_ARCH_${SWIFT_HOST_VARIANT_ARCH}_PATH}/usr/lib/swift")
+    get_property(compatibility_libs
```

Comment:
Did `add_dependencies(${target} HostCompatibilityLibs)` not work?

Reply:
Added a comment about this in https://github.com/apple/swift/commit/4d14497016780add0b674f68a7ed4594ac8ce3a9 (as this is an aspect of interface libraries that is easy to forget)